setup.py

```diff
@@ -57,7 +57,10 @@ def build_extension(self, ext):
             build_args += ["--", "/m"]
         else:
             cmake_args += ["-DCMAKE_BUILD_TYPE=" + cfg]
-            build_args += ["--", "-j{}".format(max(1, os.cpu_count() - 1))]
+            build_args += [
+                "--",
+                "-j{}".format(max(1, len(os.sched_getaffinity(0)) - 1))
```

Comment:
I think either we remove this or hardcode some small value. It's a small project, and cpu_count / affinity is not avail in old pythons

Reply:
Good points. I'll update this tomorrow. Seems like I'm having some trouble with the right Docker image, too.